### DISCUSSION TITLE
Face location not including forehead

### DISCUSSION THREAD

**Discussion by atabatic** [kex0/batch-face-swap#60] [General]
Hello, do you have advice on how to include the forehead or more of the head in my settings?
I have played with the overrides and diferrent amounts of padding in each area, but it's not seeming to do more than the face detection mask itself?

I have had good results in that regard with https://github.com/Bing-su/adetailer, but it does not support batch mode like yours.

Thank you for the advice and guidance, and your sharing this great solution!

**Comment by kex0** [comment]
Hey, I'm glad you like it.

You can play with `Mask width` and `Mask height` in `Settings`
![chrome_OfuMvVudGT](https://github.com/kex0/batch-face-swap/assets/46696708/3ba4b9e3-31fb-4b2f-a447-aabe9b6dfd9b)
